hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -3099,6 +3143,13 @@ public Builder withPayloadConfig(HoodiePayloadConfig payloadConfig) {
       return this;
     }
 
+    public Builder withRecordMergeMode(RecordMergeMode recordMergeMode) {
+      if (recordMergeMode != null) {
+        writeConfig.setValue(RECORD_MERGE_MODE, recordMergeMode.name());
```

Comment:
Is there any validation on missing `RECORD_MERGE_MODE` config, throwing an exception if missing?

Reply:
It's not needed in the write config because it has a default value